lib/puppet/transaction/additional_resource_generator.rb

```diff
@@ -51,13 +51,13 @@ def generate_additional_resources(resource)
 
   def eval_generate(resource)
     return false unless resource.respond_to?(:eval_generate)
-    raise Puppet::DevError,"Depthfirst resources are not supported by eval_generate" if resource.depthfirst?
+    raise Puppet::DevError, "Depthfirst resources are not supported by eval_generate" if resource.depthfirst?
```

Comment:
I think they're intended for users writing their own puppet code, types/providers, and functions. I.e., in this case if someone wrote a type that they intended to generate resources, they might come across this.